frontend/app/src/pages/Connected/style.scss

```diff
@@ -209,7 +209,7 @@
   }
 
   .DividerLine {
-    background-color: $connected-blue;
+    background-color: rgb(236,185,140);
```

Comment:
Let's not use hard defined colour values, but only the ones defined in `_colors.scss`

Reply:
If you need a new colour, define it there.

Reply:
Alrighty, I'll fix it asap.